release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
+* You can filter Helm charts based on their certification level and see all the Helm charts in the *Developer Catalog*.
+* The Add page now features new options to create applications and associated services and deploy them on {product-title}. These include *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, *Explore new developer features* to name a few. Users can optionally hide specific cards on the Add page.
```

Comment:
`Use the options in the *Add* page to create applications and associated services and deploy these applications and services on {product-title}.` The *Add* page options include: *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, and *Explore new developer features*.`

Note, I think `them` could be mistaken to refer to the *Add* page options. So even though it's wordier, I think it's necessary to specify that you mean the applications and services. 

I didn't know what `These include` referred to. I think it refers to the Add page options, but please confirm if I am correct. Having pronouns like 'it, this, that, there, these` as subjects of a sentence can often lead to misunderstanding, so be careful when you use them :) 

I think the last sentence can be omitted. I don't see a clear value in including it. But it's your call to include or omit it. If you do include it, consider a rewrite, something like, `Also, you can hide specific cards on the *Add* page.` 

Also, I think it should be *Add* page. s you bold *Developer Catalog* and other UI elements.